### ISSUE TITLE
WebIO not detected on OnDemand Jupyterlab 

### ISSUE THREAD

**Issue by marius311** [JuliaGizmos/WebIO.jl#507]
I'm getting WebIO not detected in a Jupyterlab instance launched by my cluster's OnDemand. The extension seems to install fine, there's even a message on the devtools console which looks like something went right to me, but I still get the not detected message. Any suggestions? Thanks. 

```
pkg> st -m WebIO IJulia
  [0f1e0344] WebIO v0.8.20
  [7073ff75] IJulia v1.24.0

julia> versioninfo()
Julia Version 1.8.5
Commit 17cfb8e65ea (2023-01-08 06:45 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 32 × Intel(R) Xeon(R) Gold 6142 CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, skylake-avx512)
  Threads: 1 on 32 virtual cores
Environment:
  JULIA_PYTHONCALL_EXE = /n/home12/marius/.cache/pypoetry/virtualenvs/muse-3g-fWYbFhxP-py3.8/bin/python
```

Jupyterlab 3.1.13. `jupyter labextension list` below. 

![webio](https://user-images.githubusercontent.com/1510968/214672930-95557ad7-f1fe-4c92-8fd7-00a1ae6f5bf4.png)

**Comment by travigd** [comment]
I don't believe the extension can be installed while JupyterLab is running -- it would need to be relaunched, which might not be possible with OnDemand.

**Comment by marius311** [comment]
Thanks. I did install the extension at the terminal, then afterwards submitted a Jupyterlab job with OnDemand, if thats what you mean.
